examples/tox21/tox21_dataset_exploration.ipynb

```diff
@@ -0,0 +1,769 @@
+    "lg.setLevel(RDLogger.CRITICAL)\n",
+    "\n",
+    "# show INFO level log from chainer chemistry\n",
+    "logging.basicConfig(level=logging.INFO)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Tox 21 dataset consists of train/validation/test data and they can be downloaded automatically with chainer chemistry. \n",
+    "The format of tox21 dataset is \"sdf\" file.\n",
+    "You can check the file path of downloaded sdf file with `get_tox21_filepath` method. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "train_filepath = datasets.get_tox21_filepath('train')\n",
+    "val_filepath = datasets.get_tox21_filepath('val')\n",
+    "test_filepath = datasets.get_tox21_filepath('test')\n",
+    "\n",
+    "print('train_filepath =', train_filepath)\n",
+    "print('val_filepath =', val_filepath)\n",
+    "print('test_filepath =', test_filepath)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Dataset contains 12 types of toxity, the label of toxity can be checked by `get_tox21_label_names` method.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "tox21 label_names = ['NR-AR', 'NR-AR-LBD', 'NR-AhR', 'NR-Aromatase', 'NR-ER', 'NR-ER-LBD', 'NR-PPAR-gamma', 'SR-ARE', 'SR-ATAD5', 'SR-HSE', 'SR-MMP', 'SR-p53']\n"
+     ]
+    }
+   ],
+   "source": [
+    "label_names = datasets.get_tox21_label_names()\n",
+    "print('tox21 label_names =', label_names)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Preprocessing dataset\n",
+    "\n",
+    "Dataset extraction depends on the preprocessing method, which is determined by `preprocessor`.\n",
+    "\n",
+    "Here, let's look an example of using `NFPPreprocessor` preprocessor for tox21 dataset exraction.\n",
+    "\n",
+    "Procedure is as follows,\n",
+    "\n",
+    "1. Instantiate `preprocessor` (here `NFPPreprocessor` is used).\n",
+    "2. call `get_tox21` method with `preprocessor`.\n",
+    " - `labels=None` option is used to extract all labels. In this case, 12 types of toxity labels are extracted (see above).\n",
+    " \n",
+    "Note that `return_smiles` option can be used to get SMILES information together with the dataset itself."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|███████████████████████████████████████████████████████████████████████████| 11764/11764 [00:22<00:00, 531.76it/s]\n",
+      "INFO:chainer_chemistry.dataset.parsers.sdf_file_parser:Preprocess finished. FAIL 0, SUCCESS 11757, TOTAL 11757\n",
```

Comment:
RDKit version also affects this result. In newer version, `SUCCESS 11757` becomes `SUCCESS 11758`. Therefore, the note about RDKit version should come before here.